docs/source/tutorial/making_predictions.rst

```diff
@@ -29,19 +30,24 @@ which will already be in memory. Each of the high-level interfaces are exposed t
 model:
 
 >>> from pykeen.pipeline import pipeline
+>>> from pykeen.models import predict
 >>> # Run the pipeline
 >>> result = pipeline(dataset='Nations', model='RotatE')
 >>> model = result.model
 >>> # Predict tails
->>> predicted_tails_df = model.get_tail_prediction_df('brazil', 'intergovorgs', triples_factory=result.training)
+>>> predicted_tails_df = predict.get_tail_prediction_df(
+...     model, 'brazil', 'intergovorgs', triples_factory=result.training)
```

Comment:
@cthoyt not sure what is the correct indention for code examples in rst

Reply:
looks right! I made a few changes to balance the trailing close parens